main.go

```diff
@@ -212,12 +211,9 @@ func runObserver(dbi *gorp.DbMap, team core.AssetManagerTeam, httpListenAddr str
 	//The observer process has a budget of 16 DB connections. Since there are
 	//much more assets than resources, we give most of these (12 of 16) to asset
 	//scraping. The rest is split between resource scrape and garbage collection.
-	goQueuedJobLoop(ctx, 12, c.PollForAssetScrapes())
-	goQueuedJobLoop(ctx, 3, c.PollForResourceScrapes())
-	go cronJobLoop(3*time.Minute, c.EnsureScrapingCounters)
```

Comment:
The counter setup is now down once at the start of `Job.Run()`, instead of continuously. The previous loop ensured that the counters were fully populated for each asset type, and since the set of asset types can change over time, this needs to be done continuously. In the new implementation, I've decided that having a single prepoulated timeseries is enough since that's enough to satisfy the absent-metrics alerts (and that's the whole usecase for prepopulating).